docker-compose.yml

```diff
@@ -1,10 +1,21 @@
-version: '3'
+version: '3.0'
 services:
-  wps:
-    build: .
-    image: pcic/sandpiper
+  sandpiper-dev:
+    image: pcic/sandpiper:latest
+    container_name: sandpiper-dev
+    environment:
+      - PYWPS_CFG=/wps.cfg
     ports:
-      - "8101:5003"
+      - "30101:5003"
```

Comment:
I believe we still will have to change the port if multiple people are working on the same bird?

Reply:
Yes that is true, but at least this way the change is minimized if there aren't multiple people working on the bird.